packages/framework/src/Model/Country/CountryDataFactory.php

```diff
@@ -28,9 +46,32 @@ public function createFromCountry(Country $country): CountryData
      * @param \Shopsys\FrameworkBundle\Model\Country\CountryData $countryData
      * @param \Shopsys\FrameworkBundle\Model\Country\Country $country
      */
-    protected function fillFromCountry(CountryData $countryData, Country $country)
+    protected function fillFromCountry(CountryData $countryData, Country $country): void
     {
-        $countryData->name = $country->getName();
+
+        /** @var \Shopsys\FrameworkBundle\Model\Country\CountryTranslation[] $translations */
+        $translations = $country->getTranslations();
+
+        foreach ($translations as $translation) {
+            $countryData->name[$translation->getLocale()] = $translation->getName();
+        }
+
+        foreach ($this->domain->getAllIds() as $domainId) {
+            $countryData->enabled[$domainId] = $country->isEnabled($domainId);
+            $countryData->priority[$domainId] = $country->getPriority($domainId);
+        }
+
         $countryData->code = $country->getCode();
     }
+
+    /**
+     * @param \Shopsys\FrameworkBundle\Model\Country\CountryData $countryData
+     */
+    protected function fillNew(CountryData $countryData): void
+    {
+        foreach ($this->domain->getAllIds() as $domainId) {
+            $countryData->enabled[$domainId] = true;
```

Comment:
I discussed it with @LukasHeinz and he prefers to have all checkbox checked (default state is taken from here).
If it were false, it would complicate DataFixtures too (need to manually enable countries).

Do you agree that it stays true?

Reply:
Yes.